generated/nidaqmx/_library_interpreter.py

```diff
@@ -4632,6 +4632,9 @@ def save_task(self, task, save_as, author, options):
             task, save_as, author, options)
         self.check_for_error(error_code)
 
+    def self_cal(self, device_name):
```

Comment:
this got generated incorrectly. I think because you still have your `functions_addon.py` changes

Reply:
Oh missed that. My bad.